test/converter/qasm_and_qiskit/test_converter_without_qiskit.py

```diff
@@ -10,7 +11,8 @@ class TestConverter(unittest.TestCase):
     How to execute this test:
```

Comment:
It is better to change the class name because this test is for qiskit.
Isn't test for all.py necessary?

Reply:
I added a test file in https://github.com/QuantestPy/quantestpy/pull/173/commits/89879f71d0718761e6f439a12e7ccdd6810cbaf9